### DISCUSSION TITLE
Chatwoot in Caprover.

### DISCUSSION THREAD

**Discussion by Toninobandolero** [caprover/caprover#1316] [Q&A] [answered]
Hello and thanks to everyone who are doing this great app.
I have finally managed to install Chatwoot on Caprover on my VPS. Everything seems to work correctly. My problem is that the chat is not fluid between chatwoot and wordpress web. **I must refresh the web** to see the new comments in the chat. My configuration may not be correct. 

Any help is welcome.

**Comment by Toninobandolero** [comment, marked as answer]
Well, i enable my https connection of chatwoot app in carover and **it works**.  Fluid chat connection. Thanks

**Reply by githubsaturn** [reply]
Glad you sorted it out. But generally, these kinds of questions are unrelated to CapRover. I suggest using Chatwood forums: https://github.com/chatwoot/chatwoot
